### ISSUE TITLE
New release request with bower support.

### ISSUE THREAD

**Issue by sillysachin** [mattkrick/lolliclock#10]
Requesting you to make a new release tag and also a bower update for 0.2.0 

0.1.0 is available @ https://libraries.io/bower/lolliclock

Also when i install using bower and use grunt i see only the .js file in the index.html file and not the .css file.

**Comment by sillysachin** [comment]
Thank for the readme.md update. Any change you will update the latest code into bower ?

**Comment by mattkrick** [comment]
Nah, IMHO I'd try to convince you to move to NPM or another package system, a nice npm/webpack workflow is 100x smoother than bower.
